### ISSUE TITLE
notification text is unreadable in gray 90 theme

### ISSUE THREAD

**Issue by elizabethsjudd** [carbon-design-system/carbon#4377]
## Environment

> Operating system

Mac

> Browser

Chrome

> Automated testing tool and ruleset

DAP: Color contrast

> Assistive technology used to verify

My eyes

## Detailed description

> What version of the Carbon Design System are you using?

10.7.1

> What did you expect to happen?

The text within low contrast notifications is readable

> What happened instead?

It text is white so it can't be read

> What WCAG 2.1 checkpoint does the issue violate?

## Steps to reproduce the issue

1. http://react.carbondesignsystem.com/?path=/story/notifications--inline
2. turn on gray 90 theme
3. turn on low contrast boolean

## Additional information

![Screen Shot 2019-10-17 at 4 33 23 PM](https://user-images.githubusercontent.com/1275779/67046588-8aa5ec80-f0fe-11e9-8c6e-c0f25c4e987e.png)
<img width="1518" alt="Screen Shot 2019-10-17 at 4 33 46 PM" src="https://user-images.githubusercontent.com/1275779/67046589-8aa5ec80-f0fe-11e9-9273-7756bf0d7ea7.png">

**Comment by emyarod** [comment]
this appears to be a duplicate of https://github.com/carbon-design-system/carbon/issues/4282
